Main.java

```diff
@@ -0,0 +1,12 @@
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {
+Ball Ball1 = Lottery.getRandomBall();
+Ball Ball2 = Lottery.getRandomBall();
+Ball Ball3 = Lottery.getRandomBall();
```

Comment:
The variable names `Ball1`, `Ball2`, and `Ball3` should start with a lowercase letter to follow Java naming conventions. Consider renaming them to `ball1`, `ball2`, and `ball3`.